language/English/strings.po

```diff
@@ -3321,7 +3321,143 @@ msgctxt "#802"
+#: xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp
+msgctxt "#803"
+msgid "Do you also want to delete all other timers from this repeating schedule?"
+msgstr ""
+
+#. Label of spinner control for choosing the timer type (once, weekdays, mondays,..)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#804"
+msgid "Schedule type:"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#805"
+msgid "Once"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#810"
+msgid "Weekdays (Mon-Fri)"
+msgstr ""
+
+#. Type selection for an advanced (epg)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#811"
+msgid "Weekends (Sat-Sun)"
+msgstr ""
+
+#. Type selection when we opened an unsupported timer with 'edit timer'
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#812"
+msgid "Unsupported repeating timer"
+msgstr ""
+
+#. Type selection for an advanced (manual)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#813"
+msgid "Mondays"
+msgstr ""
+
+#. Type selection for an advanced (manual)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#814"
+msgid "Tuesdays"
+msgstr ""
+
+#. Type selection for an advanced (manual)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#815"
+msgid "Wednesdays"
+msgstr ""
+
+#. Type selection for an advanced (manual)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#816"
+msgid "Thursdays"
+msgstr ""
+
+#. Type selection for an advanced (manual)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#817"
+msgid "Fridays"
+msgstr ""
+
+#. Type selection for an advanced (manual)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#818"
+msgid "Saturdays"
+msgstr ""
+
+#. Type selection for an advanced (manual)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#819"
+msgid "Sundays"
+msgstr ""
+
+#. Type selection for an advanced (manual)timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#820"
+msgid "All days"
+msgstr ""
+
+#. Label of new episodes button (if user don't want to record repeating episodes)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#821"
+msgid "New episodes only:"
+msgstr ""
+
+#. Label of pre-record spinner
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#822"
+msgid "Pre-record (minutes):"
```

Comment:
please note that I'm working on some label changes and with the help of @jjd-uk I renamed the related existing PVR setting to "Start padding time" and "End padding time". Your labels here should use same naming.

@ronie IMO it's a skin decision if ":" should be appended or not. That way already existing strings could be reused. Would it be possible to add the ":" in the skin? (would ofc require to change existing labels)